app/api/groups/get_permissions_can_request_help_to.feature

```diff
@@ -55,5 +69,124 @@ Feature: Get permissions can_request_help_to for a group
+      | @Chapter                    | @Class                   | @Class            | self             | false    | @HelperGroupSelf1                        |                    |
+      | @ChapterParent              | @ClassParentParentParent | @Class            | self             | false    | @HelperGroupSelfNotVisible               | without name       |
+      | @ChapterParentNoPropagation | @ClassParent             | @Class            | self             | false    | @HelperGroupSelfNoPropagation            | shouldn't appear   |
+      | @Chapter                    | @Class                   | @Class            | group_membership | false    | @HelperGroupGroupMembership1             |                    |
+      | @ChapterParent              | @ClassParent             | @Class            | group_membership | false    | @AllUsers                                |                    |
+      | @ChapterParentNoPropagation | @ClassParentParent       | @Class            | group_membership | false    | @HelperGroupGroupMembershipNoPropagation | shouldn't appear   |
+      | @Chapter                    | @ClassParent             | @Class            | item_unlocking   | false    | @HelperGroupItemUnlocking1               |                    |
+      | @ChapterParent              | @Class                   | @Class            | item_unlocking   | false    | @HelperGroupItemUnlocking2               |                    |
+      | @ChapterParentNoPropagation | @ClassParentParent       | @Class            | item_unlocking   | false    | @HelperGroupItemUnlockingNoPropagation   | shouldn't appear   |
+      | @Chapter                    | @Class                   | @Class            | other            | false    | @HelperGroupOther1                       |                    |
+      | @ChapterParent              | @ClassParentParent       | @Class            | other            | false    | @HelperGroupOther2                       |                    |
+      | @ChapterParentNoPropagation | @ClassParent             | @OtherSourceGroup | other            | false    | @HelperGroupOtherNoPropagation           | other source group |
+  # The following lines are to make the groups visible by @Teacher
+    And the group @Teacher is a descendant of the group @HelperGroupSelf1
+    And the group @Teacher is a descendant of the group @HelperGroupGroupMembership1
+    And the group @Teacher is a descendant of the group @HelperGroupGroupMembership2
+    And the group @Teacher is a descendant of the group @HelperGroupItemUnlocking1
+    And the group @Teacher is a descendant of the group @HelperGroupItemUnlocking2
+    And the group @Teacher is a descendant of the group @HelperGroupOther1
+    And the group @Teacher is a descendant of the group @HelperGroupOther2
+    When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
+    Then the response code should be 200
+    And the response at $.granted_via_self.can_request_help_to[*] should be ""
```

Comment:
`[*] should be ""` is a bit strange and difficult to interpret.
If it means "empty array", couldn't we have a rule like `And the response at $.granted_via_self.can_request_help_to should be an empty array` ?

Reply:
fixed with `[]`, but it's also possible to create a feature for it